### DISCUSSION TITLE
Support HTTPS for PostgREST Call

### DISCUSSION THREAD

**Discussion by mchatrvd** [PostgREST/postgrest#3218] [Q&A] [answered]
How can i support https for PostgREST API service? It does support http call out of the box but how can we make this to support https. Any help is appreciated. Thanks

**Comment by laurenceisla** [comment, marked as answer]
The only way to add https is via a reverse proxy like Nginx. [Check out the docs](https://postgrest.org/en/latest/explanations/nginx.html#nginx) for more info about it.
